.github/workflows/scorecards.yml

```diff
@@ -18,34 +18,40 @@ on:
 permissions: read-all
 
 jobs:
-  chainloop_init:
-    name: Chainloop Init
-    uses: chainloop-dev/labs/.github/workflows/chainloop_init.yml@64839eb68c20fefda46929c6c6e893cdf0537619
-    secrets:
-      api_token: ${{ secrets.CHAINLOOP_TOKEN }}
-    with:
-      workflow_name: "chainloop-vault-scorecards"
-      project_name: "chainloop"
-
   analysis:
     name: Scorecard analysis
     runs-on: ubuntu-latest
-    needs:
-      - chainloop_init
     permissions:
       # Needed to upload the results to code-scanning dashboard.
       security-events: write
       # Needed to publish results and get a badge (see publish_results below).
       id-token: write
       contents: read
       actions: read
+    env:
+      CHAINLOOP_WORKFLOW_NAME: "chainloop-vault-scorecards"
+      CHAINLOOP_PROJECT: "chainloop"
+      CHAINLOOP_TOKEN: ${{ secrets.CHAINLOOP_TOKEN }}
 
     steps:
+      - name: Install Cosign
```

Comment:
do we need cosign anymore? This is to verify signature of the installed chainloop correct?

Reply:
Yes, we can remove it!